src/OmniSharp.MSBuild/ProjectManager.cs

```diff
@@ -533,6 +533,7 @@ private void UpdateAnalyzerConfigFiles(Project project, IList<string> analyzerCo
         {
             if (!_workspace.EditorConfigEnabled)
             {
+                _logger.LogDebug($".editorconfig files were configured by the project {project.Name} but will not be respected because the feature is switched off in OmniSharp. Enable .editorconfig support in OmniSharp to take advantage of them.");
```

Comment:
IMO this should be Warning or at least Information.

Reply:
I am not sure. Editorconfig is off by default in OmniSharp, and I don't think we should flood with warnings when our own feature is opt-in.

Reply:
I'd agree if it was the other way round though

Reply:
According to log message above, it seems like we know that `.editoconfig` is configured by the project (do we also know if there are `.editorconfig` files found?). If that's the case, then I think we should inform the user that in order to take advantage of a feature configured by the project, then the user should also enable it in OmniSharp. In that case my opinion is that it's at least Information level.

But if this is a log message seen for every project, regardless if `.editorconfig` is intended to be used or not. Then go with debug. But maybe rephrase the log message?

Reply:
yes we know that it's configured on the project but that is still too invasive for me. update project runs many times during your work on a project so you shouldn't see the warning all the time.

If you want to have a user information mechanism then this logging is not the mechanism to do it, it's strictly for debugging purposes. The phrasing is such to aid self-debugging too.

for what you are suggesting we should just have a regular one-time message in the logs that says - editorconfig is currently disabled that is always visible.

Reply:
You are correct, because this message is shown on every Project update, it's too invasive. Didn't notice that during first review.

> for what you are suggesting we should just have a regular one-time message in the logs that says - editorconfig is currently disabled that is always visible.

Yes, this is what I was looking for initially. A single warning (or information) if a loaded project has `.editorconfig` configured, but support is disabled in OmniSharp.

Reply:
I agree with @mholo65 here.